From 65e75d43307720b2736d584c104d2502b3c9d58a Mon Sep 17 00:00:00 2001
From: Matt Porter <mporter@ti.com>
Date: Wed, 12 Sep 2012 12:29:45 -0400
Subject: [PATCH 07/10] video: st7735fb: remove FOREIGN_ENDIAN flag

Since we moved to byte swapping the framebuffer before
flushing to the spi display controller, this flag is
not longer valid. We appear as a native endian
framebuffer now.

Signed-off-by: Matt Porter <mporter@ti.com>
---
 drivers/video/st7735fb.c |    6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/drivers/video/st7735fb.c b/drivers/video/st7735fb.c
index ec003ed..ff25414 100644
--- a/drivers/video/st7735fb.c
+++ b/drivers/video/st7735fb.c
@@ -450,11 +450,7 @@ static int __devinit st7735fb_probe (struct spi_device *spi)
 	info->var.blue.length = 5;
 	info->var.transp.offset = 0;
 	info->var.transp.length = 0;
-	info->flags = FBINFO_FLAG_DEFAULT |
-#ifdef __LITTLE_ENDIAN
-			FBINFO_FOREIGN_ENDIAN |
-#endif
-			FBINFO_VIRTFB;
+	info->flags = FBINFO_FLAG_DEFAULT | FBINFO_VIRTFB;
 
 	info->fbdefio = &st7735fb_defio;
 	fb_deferred_io_init(info);
-- 
1.7.10.4

